### DISCUSSION TITLE
Spotify canvas

### DISCUSSION THREAD

**Discussion by Aussie-boii** [fast4x/RiMusic#2740] [Ideas]
Hey I wanted to ask if you can add the Spotify canvas (short video) to the app and make it a option to switch it on or off and for it to work online and offline

**Comment by fast4x** [comment]
Spotify canvas it's impossible, but in the future something maybe

**Reply by Aussie-boii** [reply]
oh I understand ok can you make a feature where you can lock the playlist cause sometimes I accidentally swipe and it removes the song and I don't know which one I removed 

Bug: The playlist has a error where setting it to title the title of the song ain't in order 
If you can please the error

**Reply by ALMIGHTY-VC** [reply]
> oh I understand ok can you make a feature where you can lock the playlist cause sometimes I accidentally swipe and it removes the song and I don't know which one I removed
> 
> Bug: The playlist has a error where setting it to title the title of the song ain't in order If you can please the error

U can create a bug template issue for it here it wont gain much importance
